### DISCUSSION TITLE
running wrong from protein-ligand complex simulation

### DISCUSSION THREAD

**Discussion by chaoyangma24** [YAMACS-SML/YAMACS#2] [General]
Thank for prompt reply. My concrete case is as follows.
I run successfully ligand or protein simulation according to the recommendation in video_tutorial, but there appeared the such following wrongs as in the following link when I run protein-ligand complex according to 3_protein_ligand_complex.webm. 

https://user-images.githubusercontent.com/83275350/229001323-028ca83f-0731-4455-a664-c6ceeab6e30a.png

**Comment by YAMACS-SML** [comment]
Thanks for sharing details about the error. The protocol you were using to perform the protein-ligand complex MD is only suitable for YASARA structure suit (Paid version of YASARA). If you notice the video (3_protein_ligand_complex.webm) carefully, then you can see a previously YASARA defined Simcell around the complex structure. YAMACS takes this Simcell size information and pass it to gromacs as a simulation cell box  size information. 
As you are using YASARA view (Free version) suite, that's why you were encountering that error. Because as per that protocol, YAMACS was looking for that predefined  Simcell by YASARA. YASARA view tool does not allow user to define the simcell around the structure. For YASARA view you need prepare the system by using the 1st, 2nd and 3rd plugins. You can not directly run the 3rd (MD run) plugin. For more information please watch the video in the following link:
https://github.com/YAMACS-SML/YAMACS/blob/main/video_tutorials/4_protein_ligand_complex_acpype.webm

Note: The video is a tutorial for protein-ligand complex system preparation using YAMACS for YASARA view suit. In this video, the ligand topology files were generated using 'Antechamber' tool of AMBER. If you have Antechamber, you can prepare the system exactly by the same procedure mentioned in the video. Alternatively, you can select 'Charrmm36' force field option, which will automatically generate the input files by using the swissparam web server.
